### ISSUE TITLE
Permissions on /var/lib/clamav-unofficial-sigs/configs/last-version-check.txt

### ISSUE THREAD

**Issue by whataboutpereira** [extremeshok/clamav-unofficial-sigs#288]
I'm getting this from cron:

`/usr/local/sbin/clamav-unofficial-sigs.sh: line 3669: /var/lib/clamav-unofficial-sigs/configs/last-version-check.txt: Permission denied`

The file was created with root permissions when I ran `clamav-unofficial-sigs.sh --force` as root like the upgrade process suggested.

**Comment by extremeshok** [comment]
what os ?

**Comment by whataboutpereira** [comment]
Sorry - its on CentOS 7.

**Comment by jpdlor** [comment]
Same on Debian buster...

**Comment by killmasta93** [comment]
same here Debian buster any ideas?

**Comment by whataboutpereira** [comment]
chown it to the same user:group as the rest of the files in the directory.

**Comment by killmasta93** [comment]
Thanks for the reply, 
Currently its like this 

```
root@mail:/var/lib/clamav-unofficial-sigs/configs# ls -l -h
total 48K
-rw-r--r-- 1 clamav clamav 7.3K Feb  7 19:56 current-dbs.txt
-rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-linuxmalwaredetect-update.txt
-rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-mbl-update.txt
-rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-ss-update.txt
-rw-r--r-- 1 clamav clamav   11 Feb  7 19:56 last-urlhaus-update.txt
-rw-r--r-- 1 clamav clamav   11 Feb  7 16:58 last-version-check.txt
-rw-r--r-- 1 clamav clamav 8.2K Feb  7 19:56 purge.txt
-rw-r--r-- 1 clamav clamav   34 Feb  4 18:45 scan-test.txt
-rw-r--r-- 1 clamav clamav 1.9K Feb  7 19:56 ss-include-dbs.txt

```
and outside of that folder

```
root@mail:/var/lib/clamav-unofficial-sigs# ls -l -h
total 916K
drwxr-xr-x 2 clamav clamav 4.0K Feb  7 19:56 configs
drwxr-xr-x 2 clamav clamav 4.0K Feb  4 18:44 dbs-add
drwxr-xr-x 2 clamav clamav 4.0K Feb  4 18:45 dbs-lmd
drwxr-xr-x 2 clamav clamav 4.0K Feb  4 18:45 dbs-mbl
drwxr-xr-x 2 clamav clamav 4.0K Feb  4 18:44 dbs-si
drwxr-xr-x 2 clamav clamav 4.0K Feb  7 18:55 dbs-ss
drwxr-xr-x 2 clamav clamav 4.0K Feb  4 18:44 dbs-yara
drwx------ 3 clamav clamav 4.0K Feb  4 18:45 gpg-key
drwxr-xr-x 2 clamav clamav 4.0K Feb  7 20:50 pid
-rw-r--r-- 1 clamav clamav 880K Feb  7 19:55 urlhaus.ndb
```

so in theory the permission are fine?

**Comment by whataboutpereira** [comment]
> Currently its like this
> 
> ```
> root@mail:/var/lib/clamav-unofficial-sigs/configs# ls -l -h
> total 48K
> -rw-r--r-- 1 clamav clamav 7.3K Feb  7 19:56 current-dbs.txt
> -rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-linuxmalwaredetect-update.txt
> -rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-mbl-update.txt
> -rw-r--r-- 1 clamav clamav   11 Feb  7 18:55 last-ss-update.txt
> -rw-r--r-- 1 clamav clamav   11 Feb  7 19:56 last-urlhaus-update.txt
> -rw-r--r-- 1 clamav clamav   11 Feb  7 16:58 last-version-check.txt
> -rw-r--r-- 1 clamav clamav 8.2K Feb  7 19:56 purge.txt
> -rw-r--r-- 1 clamav clamav   34 Feb  4 18:45 scan-test.txt
> -rw-r--r-- 1 clamav clamav 1.9K Feb  7 19:56 ss-include-dbs.txt
> ```

Judging by the timestamp it has been updated yesterday. Is it still giving the exact same error?

**Comment by Stevehans** [comment]
Had the same issue, from the instructions the first run from the installation instructins is as root, presumably the code doesn't create this file using the userid in the config file.

Not sure if that's a bug or just a step to chown missing from the install docs.

**Comment by JB1985** [comment]
No fix available in so long time?

**Comment by extremeshok** [comment]
fixed: 0a30d2d06f2d140c8f0b8cdbd3e6bedbed2bb382
